scipy/stats/_qmc.py

```diff
@@ -611,6 +612,20 @@ class QMCEngine(ABC):
     ----------
     d : int
         Dimension of the parameter space.
+    optimization : {None, "random-cd", "lloyd"}, optional
+        Whether to use an optimization scheme to improve the quality after
+        sampling.
+        Default is None.
+
+        * ``random-cd``: random permutations of coordinates to lower the
+          centered discrepancy. The best design based on the centered
+          discrepancy is constantly updated. Centered discrepancy-based
+          design shows better space filling robustness toward 2D and 3D
+          subprojections compared to using other discrepancy measures.
+        * ``lloyd``: Perturb samples using Lloyd-Max algorithm. The process
+          converges to equally spaced samples.
```

Comment:
Currently, we have "After optimization, the result would not be guaranteed to be of strength 2.", but now it would be that after optimization with method = 'lloyd', the result is no longer guaranteed to be a LHS at all, right?

Reply:
Then we should mention it.

Reply:
Done